qutebrowser/completion/models/miscmodels.py

```diff
@@ -112,6 +112,10 @@ def delete_buffer(data):
 
     model = completionmodel.CompletionModel(column_widths=(6, 40, 54))
 
+    # abbreviate tabs_are_windows for convenience
+    taw = True if config.val.tabs.tabs_are_windows else False
+    if taw: windows = []
```

Comment:
Please split this into two lines.

Reply:
why have this declaration in an if statement at all? It makes it very easy to deref the variable when it doesn't exist later on.

Reply:
@jgkamat Yeah I guess it doesn't need to be declared inside of an `if`. My original thinking was that it'd make it more explicit to readers that this list was only used when `tabs_are_windows == True`